src/drivers/infiniband/hermon.c

```diff
@@ -2162,6 +2225,17 @@ static void hermon_poll_eq ( struct ib_device *ibdev ) {
 		case HERMON_EV_PORT_STATE_CHANGE:
 			hermon_event_port_state_change ( hermon, eqe );
 			break;
+		case HERMON_EV_PORT_MGMNT_CHANGE:
+			hermon_event_port_mgmt_change ( hermon, eqe );
+			break;
+		case HERMON_EV_CQ_ERROR:
+			DBGC ( hermon, "Hermon %p EQN %#lx CQ ERROR\n",
+			       hermon, hermon_eq->eqn );
+			break;
+		case HERMON_EV_WQ_CATAS_ERROR:
+			DBGC ( hermon, "Hermon %p EQN %#lx WQ CATAS ERROR\n",
+			       hermon, hermon_eq->eqn );
+			break;
```

Comment:
I've merged the port management bits as https://github.com/ipxe/ipxe/commit/8b07c88df (after making the naming consistent between "mgmt" and "mgmnt", and matching the code style of `hermon_event_port_state_change()`).

I'm not convinced by the value of the `HERMON_EV_CQ_ERROR` and `HERMON_EV_WQ_CATAS_ERROR` handlers.  The information that they would provide is already visible in the `default` case, which also provides a hex dump to allow for more detailed debugging.

Reply:
8b07c88 lgtm!

I don't have strong feelings about HERMON_EV_CQ_ERROR / HERMON_EV_WQ_CATAS_ERROR. They were useful for debugging, just because the raw numbers are not as easy to recognize the severity of.

I'm fine with dropping these debugs.

Reply:
> I don't have strong feelings about HERMON_EV_CQ_ERROR / HERMON_EV_WQ_CATAS_ERROR. They were useful for debugging, just because the raw numbers are not as easy to recognize the severity of.

Any unrecognised event is cause for concern :)

> I'm fine with dropping these debugs.

Ack.